src/Control/Cookie_Backend.php

```diff
@@ -17,20 +17,31 @@ interface Cookie_Backend
      *
      * @param array $cookies The existing cookies to load into the cookie jar
      */
-    public function __construct($cookies = []);
+    public function __construct(array $cookies = []);
 
     /**
      * Set a cookie
      *
      * @param string $name The name of the cookie
-     * @param string $value The value for the cookie to hold
-     * @param float $expiry The number of days until expiry
-     * @param string $path The path to save the cookie on (falls back to site base)
-     * @param string $domain The domain to make the cookie available on
+     * @param string|false $value The value for the cookie to hold. Empty string or false will clear the cookie
+     * @param int $expiry The number of days until expiry; 0 means it will expire at the end of the session
+     * @param ?string $path The path to save the cookie on (falls back to site base)
+     * @param ?string $domain The domain to make the cookie available on
```

Comment:
```suggestion
     * @param null|string $path The path to save the cookie on (falls back to site base)
     * @param null|string $domain The domain to make the cookie available on
```
Don't think I've seen `?` in a docblock param before, should be consistent

Reply:
Done